script_library/requirements.txt

```diff
@@ -4,3 +4,5 @@ jmespath==0.9.3
 requests==2.21.0
 ara==0.16.3
 openstacksdk==0.22.0
+python-openstackclient
```

Comment:
So the commit message doesn't explain *why* we need those. normally the ansible scripts are using openstacksdk directly, why do we suddenly require the clients?

Reply:
I don't think it's sudden, but we have ansible command modules calling the openstack cli in `playbooks/openstack-create_network.yml` and `playbooks/roles/airship-deploy-tempest/tasks/main.yml`